---
title: "Daniel R. Kick's resume"
author: Daniel Kick
Date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    css: 
      - css/custom_resume.css
      - css/styles.css
      - resume
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
---

```{r eval=FALSE, include=FALSE}
icons::icon_find('users', set = "fontawesome")
icons::icon_find('leader')
```
<!-- Formatting Ref: -->

<!-- These breaks just force a new page so section doesn't get cut off -->
<!-- <br> -->
<!-- <br> -->
<!-- <br> -->
<!-- <span style="color: red;">text</span> -->
<!-- <font size="+2">This is bigger text.</font> -->
<!-- <font size="-1">This is smaller text.</font> -->
<!-- <font size="1">This is really tiny text.</font> -->

<!-- this controls the icon positioning -->
<!-- --decorator-horizontal-margin: .2in;  -->
```{css, echo=FALSE}
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.22in; 
}
```

```{r, include=FALSE}
knitr::opts_chunk$set(
  results='asis', 
  echo = FALSE
)
# Useful Refs
# https://annekelincoln.com/resume-in-r/
# https://shaziaruybal.github.io/automate-cv-rmd/
# https://github.com/amalasi2418/Blog-post/tree/master/resume/Automate



library(magrittr) # For the pipe
source("cv_printing_functions.r")


library(tidyverse)
# Read in all data and initialize a CV printer object
position_data <- read_csv("csvs/positions.csv")
# skills        <- read_csv("csvs/language_skills.csv")
# text_blocks   <- read_csv("csvs/text_blocks.csv")
# contact_info  <- read_csv("csvs/contact_info.csv", skip = 1)



library(glue) # for glue_data()


# Is the goal of this knit to build a document that is exported to PDF? If so
# set this to true to have links turned into footnotes at the end of the
# document
PDF_EXPORT <- FALSE


# A global (gasp) variable that holds all the links that were inserted for
# placement at the end
links <- c()

# ======================================================================
# Now we source two external scripts. One contains functions for building the
# text output and the other loads up our data from either googlesheets or csvs

# Functions for building sections from CSV data
source('parsing_functions.R')

# Load data for CV/Resume
# source('gather_data.R')

# Now we just need to filter down the position data to include less verbose
# categories and only the entries we have designated for the resume
position_data <- position_data %>% 
  filter(in_resume) %>% 
  mutate(
    # Build some custom sections by collapsing others
    section = case_when(
      section %in% c('research_positions', 'industry_positions') ~ 'positions', 
      section %in% c('data_science_writings', 'by_me_press') ~ 'writings',
      TRUE ~ section
    )
  ) 




```

Aside
================================================================================
### Contact Info.

- `r icons::fontawesome$solid$envelope` [hello@danielkick.com](mailto:hello@danielkick.com) 
- `r icons::fontawesome$brands$linkedin` [daniel-kick-5a449b9a](https://www.linkedin.com/in/daniel-kick-5a449b9a/)
- `r icons::academicons$"google-scholar"` [Google Scholar](https://scholar.google.com/citations?user=5Sz0E-0AAAAJ&hl=en&oi=ao)
<!-- - `r icons::fontawesome$brands$orcid` [0000-0002-9002-1862](https://orcid.org/0000-0002-9002-1862) -->
- `r icons::fontawesome$brands$orcid` [ORCID](https://orcid.org/0000-0002-9002-1862)
- `r icons::fontawesome$brands$github` [github.com/danielkick](https://github.com/danielkick)
- `r icons::fontawesome$solid$link` [danielkick.com](https://www.danielkick.com)

## Education
### PhD: Biological Sciences

University of Missouri, Columbia, MO (2021)

<font size="-2">**Machine Learning Methods for Biomedical Informatics**, Quantitative Methods in the Life Sciences, and Grant Writing</font> 

<br>

### Bachelor of Science: Biology

Truman State University, Kirksville, MO (2015)

<font size="-2">Next Generation Sequence Data and Analysis, Bioinformatics, ANOVA and Experimental Design, Non-Parametric Statistics

**Leadership role** in the biological honors society Tri-Beta</font>


<br>

### Technical Skills

 `r icons::fontawesome$brands$"r-project"`: **R **Programming  (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2017-01-29"), units = "weeks"))/52, 0)` years) experience with <span style="color: DarkRed;">`tidyverse`, `lme4`, `caret`, `ggplot2`, `shiny`</span>, & package creation. 
 
 
 `r icons::fontawesome$brands$python`: **Python **Programming  (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2021-06-09"), units = "weeks"))/52, 0)` years) experience with <span style="color:DarkRed;">`pandas`, `numpy`, `plotly`, `scikit-learn`, `keras`, `pytorch`</span>.
 

 `r icons::octicons$terminal`: **Miscellaneous **
 Experience with high performance computing (<span style="color:DarkRed;">`bash`, `slurm`</span>), virtual environments (<span style="color:DarkRed;">`conda`, `singularity`, `docker`</span>), version control (<span style="color:DarkRed;">`git`, `GitHub`</span>), literate programming (<span style="color:DarkRed;">`Rmarkdown`, `Jupyter`</span>).


<!-- `r icons::octicons$person`: **Soft Skills** Adaptability | Problem Solving | Self-Directed Learning | Leadership | Mentorship  -->


Disclaimer {#disclaimer}
--------------------------------------------------------------------------------
For a pdf with

links scan here.



```{r, out.width="75%"}
knitr::include_graphics("./qr_gh_Resume_Resume.gif")
```
<!-- This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown). -->
<!-- Last updated on `r Sys.Date()`. -->


Main
================================================================================

Daniel R. Kick, PhD {#title}
--------------------------------------------------------------------------------

- Experience with **statistics, programming, machine learning, and deep learning**.
- **Presented** to scientific and general audiences **>25** times since 2016. 
- **Led** 4 teaching assistants and **mentored** 7 research students.
- **Developed statistical tool** used by >700 students as of 2021.

```{r}
# paste(c("a", "b"), collapse = " | ")
```

<!-- Problem-solving |  Detail & Results-oriented | Statistical Analysis | Data Visualization | -->
<!-- Literate Programming | Technical & Non-technical Writer | Research & Development | Public Speaking | -->
<!-- Regulatory Affairs | Experimental Design | Method Development | Training, Mentorship, & Recruitment | -->
<!-- Collaborative | Open-mindedness | Growth mindset | Resourceful | Dependability | Adaptability | Self-starter | -->
<!-- Leadership | Effective Communicator | Fluorescent Microscopy | Microbiology | Pharmacology | Life Sciences -->


Professional and Research Experience {data-icon=laptop}
--------------------------------------------------------------------------------
### Research Geneticist

Jacob Washburn Lab, USDA-ARS

N/A

Present - 2021

- Employed **deep neural networks, machine learning models,** and **best linear unbiased predictors** to **improve corn yield prediction accuracy** in diverse environments (see *Kick et al., 2023*).
- **Designed project plan** and **submitted grant** application titled **"Environmentally Aware Deep Learning Based Genomic Selection And Management Optimization For Maize Yield"** to the NIFA Agriculture and Food Research Initiative's Education and Workforce Development Program (*Decision Pending*).
- **Communicated** with stakeholders through **8 presentations** <font size="1">(4 national, 2 regional, and 2 outreach presentations)</font>. 
- **Mentored** 2 students conducting a high throughput root phenotyping experiment and wrote scripts for data organization and analysis. 
- **Created** and taught a **Python data visualization workshop** titled "Tools and Techniques for a Jupyter Based Scientific Workflow".
- **Collaborated with plant biologists**, contributing statistical expertise (manuscript in preparation).
- Completed **Software Carpentries instructor certification**, taught **R for Reproducible Scientific Analysis**, and assisted in teaching **Data Management with SQL**.
- Designed and completed a professional development curriculum with the guidance of an industry scientist via the **Bayer-University Mentoring Program**.
- Served as a **panel member on "Next-Generation Omics"** at the 2022 University of Missouri Division of Biological Sciences Retreat.


### Graduate Researcher

David Schulz Lab, University of Missouri

N/A

2021 - 2016

- **Author on 6 publications**<font size="1">: 4 original research and 2 eLife Insight publications.</font>
- <span style="color: #003399;">**Assessed the efficacy of machine learning models** to identify cell identity from mRNA and contig abundances.</span> **Applied cluster estimation, hyperparameter tuning, unsupervised machine learning,** and **supervised machine learning**. Identified and learned needed skills primarily through self study. Collaborated with molecular biology project lead. (see *Northcutt^1^, Kick^1^, et al. 2019*).
- <span style="color: #003399;">Demonstrated activity desynchronization induces changes in neuronal connections.</span> Defined research question and experiments. **Developed novel approach to quantify changes in cell activity** (see *Kick and Schulz 2022*). 
- <span style="color: #003399;">Investigated activity dependent changes in neuronal excitability, conductances, and ion channel mRNA abundances.</span> Designed experiments, collected data, performed analysis, and **developed novel method for quantifying changes in cell activity** using *in silico* simulations.
- **Collaborated with electrophysiologists**, assisting with data organization, cleaning, and analysis.
- **Collaborated with computational neuroscientists**, contributing biological and statistical expertise (in preparation).
- **Mentored 5 students** and oversaw their projects. 
- Communicated results through **17 presentations** <font size="1">(6 national, 6 regional, and 5 outreach and recruitment)</font>. 
- Served as a **peer mentor of 3 PhD students** in use of R for reproducible data analysis, created internal documents on same.

<!-- (https://www.pnas.org/doi/10.1073/pnas.1911413116) -->
<!-- [2](https://www.jneurosci.org/content/42/35/6751) -->
<!-- [3](https://mospace.umsystem.edu/xmlui/handle/10355/85790) -->

### Lead Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2021 - 2020

- **Developed statistics web application used by more than 700 students** as of 2021 with **shiny** ([source](https://github.com/danielkick/MultiMethodPhysApp), [deployed](https://danielkick.shinyapps.io/BioSc3700_1)) for data visualization, testing assumptions, and fitting **frequentist**, **non-parametric**, and **Bayesian** models.
- Led 4 Teaching Assistants and coordinated adaptation of lab curriculum to be fully online due to COVID-19 pandemic.
- **Mentored** next Lead Teaching Assistant, created documentation on best practices.

### Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2020 - 2018

- Updated and refined curriculum.
- Delivered lectures and ensured experiments were conducted safely.
- Modeled student grade distributions to identify and adjust for differences in grading.


### Curriculum Consultant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2018

- Redesigned course material to incorporate primary literature and data analysis.


### Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2016 - 2015

- Delivered weekly lectures, ensured experiments were conducted safely, provided timely feedback on assignments.


### Undergradute Researcher

University of Missouri, University of Connecticut, and Truman State University

N/A

2013 - 2015

- Designed a hydroponic system for maize root phenotyping <font size="1">-- Diane Janick-Buckner and Brent Buckner, Truman State University (2014-2015)</font>, Quantified retinal minor splicisome expression using immunohistochemistry <font size="1">-- (NSF REU) Rahul Kanadia, University of Connecticut (2014)</font>, Measured effectiveness of oligonucleotide treatment for spinal muscular atrophy in mice <font size="1">-- (NSF REU) Christian Lorson, University of Missouri (2013)</font>.



Honors and Awards {data-icon=award}
--------------------------------------------------------------------------------

### Poster Ranked First in University of Missouri Plant Research Symposium Poster Competition

N/A

N/A

2022


### J. Perry Gustafson Award for Outstanding Graduate Research in the Life Sciences

[This award](https://randall-travel-award.missouri.edu/gustafson-award/) is granted for the quality of a student's **independent research and academic achievements**. Recipients receive a \$2,000 award.

N/A

2019


### National Institutes of Health T32 Training Grant Recipient

This fellowship provides a \$27,000 yearly stipend and travel awards of \$750.  

N/A

2018 - 2016


### Cum Laude & President's Recognition, Truman State University

N/A

N/A

2015



Selected Publications (3/7, +1 in Review, +2 in Prep.) {data-icon=flask}
--------------------------------------------------------------------------------
```{r}
# position_data %>% print_section('academic_articles')
```
### Yield Prediction Through Integration of Genetic, Environment, and Management Data Through Deep Learning

<font size="1">***Daniel R. Kick***, Jason G. Wallace, James C. Schnable, Judith M. Kolkman, Baris Alaca, Timothy M. Beissinger, David Ertl, Sherry Flint-Garcia, Joseph L. Gage, Candice N. Hirsch, Joseph E. Knoll, Natalia de Leon, Dayane C. Lima, Danilo Moreta, Maninder P. Singh, Teclemariam Weldekidan, Jacob D. Washburn [G3: Genes, Genomes, Genetics](https://academic.oup.com/g3journal/advance-article/doi/10.1093/g3journal/jkad006/6982634)</font>

N/A

2023


### Timing dependent potentiation and depression of electrical synapses contributes to network stability in the crustacean cardiac ganglion

<font size="1">***Daniel R. Kick*** and David J. Schulz [The Journal of Neuroscience](https://www.jneurosci.org/content/42/35/6751)</font>

N/A

2022
<!-- [Code](https://github.com/danielkick/ElectricalSynapsePotentiation) -->


### Molecular profiling of single neurons of known identity in two ganglia from the crab Cancer borealis

<font size="1">Adam J. Northcutt^1^, ***Daniel R. Kick^1^***, Adriane G. Otopalik, Benjamin M. Goetz, Rayna M. Harris, Joseph M. Santin, Hans A. Hofmann, Eve Marder, and David J. Schulz (***^1^ denotes co-first authorship***) [Proceedings of the National Academy of Sciences](https://www.pnas.org/doi/10.1073/pnas.1911413116)</font>

N/A

2019


<!-- [Code](https://github.com/danielkick/STGmRNACellClassification) -->

