---
title: "Daniel R. Kick's resume"
author: Daniel Kick
Date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    css: 
      - css/custom_resume.css
      - css/styles.css
      - resume
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
---

```{css, echo=FALSE}
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}
```

```{r, include=FALSE}
knitr::opts_chunk$set(
  results='asis', 
  echo = FALSE
)
# Useful Refs
# https://annekelincoln.com/resume-in-r/
# https://shaziaruybal.github.io/automate-cv-rmd/
# https://github.com/amalasi2418/Blog-post/tree/master/resume/Automate



library(magrittr) # For the pipe
source("cv_printing_functions.r")


library(tidyverse)
# Read in all data and initialize a CV printer object
position_data <- read_csv("csvs/positions.csv")
skills        <- read_csv("csvs/language_skills.csv")
text_blocks   <- read_csv("csvs/text_blocks.csv")
contact_info  <- read_csv("csvs/contact_info.csv", skip = 1)



library(glue) # for glue_data()


# Is the goal of this knit to build a document that is exported to PDF? If so
# set this to true to have links turned into footnotes at the end of the
# document
PDF_EXPORT <- FALSE


# A global (gasp) variable that holds all the links that were inserted for
# placement at the end
links <- c()

# ======================================================================
# Now we source two external scripts. One contains functions for building the
# text output and the other loads up our data from either googlesheets or csvs

# Functions for building sections from CSV data
source('parsing_functions.R')

# Load data for CV/Resume
# source('gather_data.R')

# Now we just need to filter down the position data to include less verbose
# categories and only the entries we have designated for the resume
position_data <- position_data %>% 
  filter(in_resume) %>% 
  mutate(
    # Build some custom sections by collapsing others
    section = case_when(
      section %in% c('research_positions', 'industry_positions') ~ 'positions', 
      section %in% c('data_science_writings', 'by_me_press') ~ 'writings',
      TRUE ~ section
    )
  ) 




```
 


Aside
================================================================================

```{r}
knitr::include_graphics("Headshot.png")
```


Contact Info {#contact}
--------------------------------------------------------------------------------

- `r icons::fontawesome$solid$envelope` [hello@danielkick.com](hello@danielkick.com)
- `r icons::fontawesome$solid$link` [danielkick.com](https://www.danielkick.com)
- `r icons::fontawesome$brands$github` [github.com/danielkick](https://github.com/danielkick)
- `r icons::fontawesome$brands$linkedin` [daniel-kick-5a449b9a/](https://www.linkedin.com/in/daniel-kick-5a449b9a/)
- `r icons::academicons$"google-scholar"` [Google Scholar](https://scholar.google.com/citations?user=5Sz0E-0AAAAJ&hl=en&oi=ao)
- `r icons::fontawesome$brands$orcid` [0000-0002-9002-1862](https://orcid.org/0000-0002-9002-1862)



```{r eval=FALSE, include=FALSE}
icons::icon_find('users', set = "fontawesome")
icons::icon_find('keras')
```





<!-- These breaks just force a new page so section doesn't get cut off -->
<!-- <br> -->
<!-- <br> -->
<!-- <br> -->

Disclaimer {#disclaimer}
--------------------------------------------------------------------------------
For a pdf with

links scan here.

```{r, out.width="75%"}
knitr::include_graphics("./qr_gh_Resume_Resume.gif")
```


<!-- This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown). -->

<!-- Last updated on `r Sys.Date()`. -->


Main
================================================================================

Daniel R. Kick, PhD {#title}
--------------------------------------------------------------------------------

I am a Postdoctoral researcher working at the USDA Agricultural Research Service (ARS). Professionally my goal is to support human food security by leveraging statistical and computational tools to improve crop resiliency and yield. My current research aims to use deep learning to more accurately predict crop yields using genetic, environmental, and management data. This work focuses on corn (*Zea mays*) while being extendable to other row crops. During my PhD, I applied machine learning to identify neuronal cell types from transcriptomic and ion channel mRNA abundance data. I also applied statistical modeling in conjunction with electrophysiological experiments to study activity dependent compensation in neurons and neural circuits.

### Competencies

I have experience with statistical modeling, machine learning, deep learning, experimental design, and data visualization & communication. 

 `r icons::fontawesome$brands$"r-project"`: R Programming (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2017-01-29"), units = "weeks"))/52, 1)` years) including experience with `tidyverse`, `lme4`, `caret`, `ggplot2`, `shiny`, & package creation. 
 
 `r icons::fontawesome$brands$python`: Python Programming (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2021-06-09"), units = "weeks"))/52, 1)` years) including experience with `pandas`, `plotly`, `scikit-learn`, `keras`.

 `r icons::octicons$terminal`: Experience high performance computing (`bash`, `slurm`), virtual environments (`conda`, `singularity`, `docker`), version control (`git`), literate programming (`Rmarkdown`, `Jupyter`).


<!-- Education {data-icon=graduation-cap data-concise=true} -->

Relevant Research Experience {data-icon=flask}
--------------------------------------------------------------------------------
### Research Geneticist

Jacob Washburn Lab, USDA-ARS

N/A

Present - 2021

- Improved yield prediction of maize in diverse environments by using deep learning to better capture gene by environment effects. Conducted research and communicated results to stakeholders through local and national presentations, and a scientific publication (in review). Supervised undergraduates conducting a high throughput phenotyping project and assisted with wrote scripts for data organization and analysis.



### Graduate Student

David Schulz Lab, Missouri State University

N/A

2021 - 2016

- Assessed the efficacy of machine learning models to recapitulate neural cell identity from mRNA expression [1](https://www.pnas.org/doi/10.1073/pnas.1911413116). Demonstrated that activity desynchronization induces degree dependent changes in conductance between neurons [2](https://www.jneurosci.org/content/42/35/6751). Investigated the compensatory effects of elevated depolarization on neuronal excitability, conductances, and ion channel mRNA abundances in small neural networks [3](https://mospace.umsystem.edu/xmlui/handle/10355/85790).

Selected Teaching Experiences {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------
### Workshop Creator; Tools and Techniques for a `Jupyter` Based Scientific Workflow

Created and delivered a workshop on data visualization in Python for University of Missouri [Bioinformatics in Plant Science](https://undergradresearch.missouri.edu/jobs-and-programs/programs/bioinformatics-in-plants-bips/)

N/A

2022

 

### [Software Carpentries](https://carpentries.org/) Certified Instructor

Received theoretical and practical instruction on leading computational workshops. Taught [R for Reproducible Scientific Analysis](https://swcarpentry.github.io/r-novice-gapminder/), assisted in teaching [Data Management with SQL](http://datacarpentry.org/sql-ecology-lesson/).

N/A

2022

<!-- ### Mentoring Undergraduate Researchers -->

<!-- As a **Postdoc: Supervised 2 students** conducting a high throughput root phenotyping experiment: Grace Sidberry (2021-pres.),  Madi Michell (2022-pres.). As a **PhD Student: Trained 5 students** in electrophysiolgy techniques and oversaw their projects. Abby Beckerdite (2016-2019), Ayla Ross (2019), Katlyn Sullivan (2018), Kelly Hiersche (2017), & Rody Kingston (2016) -->

<!-- N/A -->

<!-- 2022 - 2016 -->

 


### Lead Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2021 - 2018

- While lead TA (2020-2021) I coordinated adaptation and expansion of lab material to be fully online due to Covid-19. Developed and deployed a statistics web application *used by a minimum 705 students* as of 2021 [source](https://github.com/danielkick/MultiMethodPhysApp) , [deployed](https://danielkick.shinyapps.io/BioSc3700_1). Includes capability for visualization, testing model assumptions, frequentist models, non-parametric tests, basic Bayesian models.


<!-- ### Teaching Assistant, Animal Physiology Lab -->

<!-- Biological Sciences, University of Missouri -->

<!-- N/A -->

<!-- 2020 - 2018 -->

<!-- - Tested additional curriculum alterations, tested grade distributions to identify and adjust for grader effects. -->


<!-- ### Curriculum Consultant, Animal Physiology Lab -->

<!-- Biological Sciences, University of Missouri -->

<!-- N/A -->

<!-- 2018 -->

<!-- - Updated curriculum and redesigned experiments placing a greater focus primary literature and data analysis. -->


<!-- ### Teaching Assistant, Animal Physiology Lab -->

<!-- Biological Sciences, University of Missouri -->

<!-- N/A -->

<!-- 2016 - 2015 -->

<!-- - Provided weekly lectures on relevant background, ensured experiments were conducted safely, provided timely feedback on assignments. -->



Selected Publications {data-icon=file}
--------------------------------------------------------------------------------
```{r}
# position_data %>% print_section('academic_articles')
```
### Yield Prediction Through Integration of Genetic, Environment, and Management Data Through Deep Learning

*Daniel R. Kick*, Jason G. Wallace, James C. Schnable, Judith M. Kolkman, Baris Alaca, Timothy M. Beissinger, David Ertl, Sherry Flint-Garcia, Joseph L. Gage, Candice N. Hirsch, Joseph E. Knoll, Natalia de Leon, Dayane C. Lima, Danilo Moreta, Maninder P. Singh, Teclemariam Weldekidan, Jacob D. Washburn

[bioRxiv (Under-Review)](https://www.biorxiv.org/content/10.1101/2022.07.29.502051v1)

2022

 


### Timing dependent potentiation and depression of electrical synapses contributes to network stability in the crustacean cardiac ganglion

*Daniel R. Kick* and David J. Schulz

[The Journal of Neuroscience](https://www.jneurosci.org/content/42/35/6751)

2022

 


### Molecular profiling of single neurons of known identity in two ganglia from the crab Cancer borealis

*Adam J. Northcutt^1^, Daniel R. Kick^1^*, Adriane G. Otopalik, Benjamin M. Goetz, Rayna M. Harris, Joseph M. Santin, Hans A. Hofmann, Eve Marder, and David J. Schulz (**^1^** denotes co-first authorship)

[Proceedings of the National Academy of Sciences](https://www.pnas.org/doi/10.1073/pnas.1911413116)

2019


<!-- ### Yield Prediction Through Integration of Genetic, Environment, and Management Data Through Deep Learning -->

<!-- [bioRxiv (Under-Review)](https://www.biorxiv.org/content/10.1101/2022.07.29.502051v1) -->

<!-- N/A -->

<!-- 2022 -->

<!-- - Daniel R. Kick, Jason G. Wallace, James C. Schnable, Judith M. Kolkman, Barış Alaca, Timothy M. Beissinger, David Ertl, Sherry Flint-Garcia, Joseph L. Gage, Candice N. Hirsch, Joseph E. Knoll, Natalia de Leon, Dayane C. Lima, Danilo Moreta, Maninder P. Singh, Teclemariam Weldekidan, Jacob D. Washburn -->



<!-- ### Timing dependent potentiation and depression of electrical synapses contributes to network stability in the crustacean cardiac ganglion -->

<!-- [The Journal of Neuroscience](https://www.jneurosci.org/content/42/35/6751) -->

<!-- [Code](https://github.com/danielkick/ElectricalSynapsePotentiation) -->

<!-- 2022 -->

<!-- - Daniel R. Kick and David J. Schulz -->



<!-- ### Molecular profiling of single neurons of known identity in two ganglia from the crab Cancer borealis -->

<!-- [Proceedings of the National Academy of Sciences](https://www.pnas.org/doi/10.1073/pnas.1911413116) -->

<!-- [Code](https://github.com/danielkick/STGmRNACellClassification) -->

<!-- 2019 -->

<!-- - Adam J. Northcutt1, Daniel R. Kick1, Adriane G. Otopalik, Benjamin M. Goetz, Rayna M. Harris, Joseph M. Santin, Hans A. Hofmann, Eve Marder, and David J. Schulz -->



<!-- ### Cell Communication: Studying gap junctions with PARIS -->

<!-- [eLife](https://elifesciences.org/articles/45207) -->

<!-- N/A -->

<!-- 2019 -->

<!-- - Daniel R Kick, David J Schulz -->



<!-- ### Dopamine maintains network synchrony via direct modulation of gap junctions in the crustacean cardiac ganglion -->

<!-- [eLife](https://elifesciences.org/articles/39368) -->

<!-- N/A -->

<!-- 2018 -->

<!-- - Brian J Lane, Daniel R Kick, David K Wilson, Satish S Nair, David J Schulz -->



<!-- ### Motor Systems: Variability in neural networks -->

<!-- [eLife](https://elifesciences.org/articles/34153) -->

<!-- N/A -->

<!-- 2018 -->

<!-- - Daniel R Kick, David J Schulz -->



<!-- ### The Hillary Climber trumps manual testing: an automatic system for studying Drosophila climbing -->

<!-- [The Journal of Neurogenetics](https://www.tandfonline.com/doi/abs/10.1080/01677063.2016.1255211) -->

<!-- N/A -->

<!-- 2016 -->

<!-- - Alex M. Willenbrink, Margo K. Gronauer, Leon F. Toebben, Daniel R. Kick, Madalyn Wells & Bing Zhang -->
<!-- - Journal of Neurogenetics -->











Selected Presentations {data-icon=users}
--------------------------------------------------------------------------------
### Improving Maize Yield Prediction through Genetic, Environmental, and Management Factor Interactions with Deep Learning

_ASA, CSSA and SSSA International Annual Meetings, Baltimore MD

N/A

2022

- *Daniel R Kick*, Jacob D. Washburn (2022)
- Poster


### Yield Prediction Accuracy is Improved Through Incorporating Genetic, Environmental, and Management Interactions with Deep Learning,

University of Missouri Interdisciplinary Plant Group Symposium, Columbia MO

N/A

2022

- Daniel R Kick, Jacob D. Washburn (2022)
- Oral Presentation


### Yield Prediction Through Integration of Genetic, Environment, and Management Data by Deep Learning

University of Missouri MU Plant Research Symposium, Columbia MO

N/A

2022

- Daniel R Kick, Jacob D Washburn
- *Ranked first in poster competition*


<!-- These breaks just force a new page so section doesn't get cut off --> 
<!-- <br> -->
<!-- <br> -->
<!-- <br> -->

Professional Activities {data-icon=suitcase}
--------------------------------------------------------------------------------
### Panel Member, Next-Generation Omics, Biological Sciences Divisional Retreat

Ruthie Angelovici, David J Schulz, Daniel R Kick, and Mannie Liscum, University of Missouri Division of Biological Sciences Retreat

N/A

2022







Honors and Awards  {data-icon=award}
--------------------------------------------------------------------------------
### J. Perry Gustafson Award for Outstanding Graduate Research in the Life Sciences

[This award](https://randall-travel-award.missouri.edu/gustafson-award/) is granted in recognition of the quality of their independent research and academic achievements. Recipients receive a $2,000 award.

N/A

2019

 


### National Institutes of Health T32 Training Grant Recipient

This fellowship provides a $27,000 yearly stipend and two travel awards of $750 to facilitate presenting research at scientific conferences.

N/A

2018 - 2016




Education {data-icon=graduation-cap}
--------------------------------------------------------------------------------
### PhD. Biological Sciences

University of Missouri, Columbia, MO

N/A

2021 - 2015

- Coursework included Machine Learning Methods for Biomedical Informatics, Quantitative Methods in the Life Sciences, and Grant Writing


### Bachelor of Science

Truman State University, Kirksville, MO

N/A

2015 - 2011

- Coursework included Next Generation Sequence Data and Analysis, Bioinformatics, Analysis of Variance and Experimental Design, Non-Parametric Statistics, and Economic & Medicinal Botany

```{r}
# position_data %>% print_section('education')
```
 <!-- {data-icon=laptop} -->
 
 
