---
title: "Daniel R. Kick's resume"
author: Daniel Kick
Date: "`r Sys.Date()`"
output:
  pagedown::html_resume:
    css: 
      - css/custom_resume.css
      - css/styles.css
      - resume
    # set it to true for a self-contained HTML page but it'll take longer to render
    self_contained: true
# uncomment this line to produce HTML and PDF in RStudio:
knit: pagedown::chrome_print
---



```{r eval=FALSE, include=FALSE}
icons::icon_find('users', set = "fontawesome")
icons::icon_find('leader')
```
<!-- Formatting Ref: -->

<!-- These breaks just force a new page so section doesn't get cut off -->
<!-- <br> -->
<!-- <br> -->
<!-- <br> -->
<!-- <span style="color: red;">text</span> -->
<!-- <font size="+2">This is bigger text.</font> -->
<!-- <font size="-1">This is smaller text.</font> -->
<!-- <font size="1">This is really tiny text.</font> -->

<!-- this controls the icon positioning -->
<!-- --decorator-horizontal-margin: .2in;  -->
```{css, echo=FALSE}
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 0rem;
  --sidebar-background-color: #ffffff;
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.22in; 
}
```

```{r, include=FALSE}
knitr::opts_chunk$set(
  results='asis', 
  echo = FALSE
)
# Useful Refs
# https://annekelincoln.com/resume-in-r/
# https://shaziaruybal.github.io/automate-cv-rmd/
# https://github.com/amalasi2418/Blog-post/tree/master/resume/Automate



library(magrittr) # For the pipe
source("cv_printing_functions.r")


library(tidyverse)
# Read in all data and initialize a CV printer object
position_data <- read_csv("csvs/positions.csv")
position_data_all <- position_data
# skills        <- read_csv("csvs/language_skills.csv")
# text_blocks   <- read_csv("csvs/text_blocks.csv")
# contact_info  <- read_csv("csvs/contact_info.csv", skip = 1)



library(glue) # for glue_data()


# Is the goal of this knit to build a document that is exported to PDF? If so
# set this to true to have links turned into footnotes at the end of the
# document
PDF_EXPORT <- FALSE


# A global (gasp) variable that holds all the links that were inserted for
# placement at the end
links <- c()

# ======================================================================
# Now we source two external scripts. One contains functions for building the
# text output and the other loads up our data from either googlesheets or csvs

# Functions for building sections from CSV data
source('parsing_functions.R')

# Load data for CV/Resume
# source('gather_data.R')

# Now we just need to filter down the position data to include less verbose
# categories and only the entries we have designated for the resume
position_data <- position_data %>% 
  filter(in_resume) %>% 
  mutate(
    # Build some custom sections by collapsing others
    section = case_when(
      section %in% c('research_positions', 'industry_positions') ~ 'positions', 
      section %in% c('data_science_writings', 'by_me_press') ~ 'writings',
      TRUE ~ section
    )
  ) 
```



```{r}
# helper function to update fractions of entries in resume/ in cv
resume_frac <- function(get_section = '', half = FALSE){
  chr_count <- function(df){
          return(as.character(as.numeric(tally(df))))
  }
  
  mk_standard <- function(get_section, half = FALSE){
    if(!half){
      return(paste0(
        chr_count(filter(position_data, section == get_section)),
        '/',
        chr_count(filter(position_data_all, section == get_section))
      ))          
    } else {
      return(
        chr_count(filter(position_data_all, section == get_section))
      )
    }
  }
  
  if(get_section == 'honors'){
    output <- mk_standard(get_section = get_section, half = half)
    
  }else if(get_section == 'academic_articles'){
    output <- mk_standard(get_section = get_section, half = half)
    
  }else if(get_section == 'academic_articles_review'){
    if(!half){
      output <- paste0(
        chr_count(position_data[(
          position_data$section == get_section &
          stringr::str_detect(position_data[['loc']], pattern = '(In Review)') &
          !is.na(position_data[['loc']])  ), ] ),
        '/',
        chr_count(position_data_all[(
          position_data_all$section == get_section &
          stringr::str_detect(position_data_all[['loc']], pattern = '(In Review)') &
          !is.na(position_data_all[['loc']])  ), ] )
      )    
    } else {
      output <- chr_count(position_data_all[(
          position_data_all$section == get_section &
          stringr::str_detect(position_data_all[['loc']], pattern = '(In Review)') &
          !is.na(position_data_all[['loc']])  ), ] )
    }
    
  }else if(get_section == 'academic_articles_prep'){
      output <- mk_standard(get_section = get_section, half = half)    
    
  }else{
    output <- '<span style="color: red;">MISSING</span>'
  }
  return(output)

}

# resume_frac(get_section = 'honors')
```

```{r}
row_count <- function(df){
          return(as.numeric(tally(df)))
}

all_pres <- row_count(filter(position_data_all, 
                             section %in% c('national_presentations', 
                                            'local_presentations',
                                            'outreach_presentations'), end >= 2016))


postdoc_pres_national <- row_count(filter(position_data_all, 
                                          section == 'national_presentations', end >= 2021))
postdoc_pres_local    <- row_count(filter(position_data_all, 
                                          section == 'local_presentations', end >= 2021))
postdoc_pres_outreach <- row_count(filter(position_data_all, 
                                          section == 'outreach_presentations', end >= 2021))

phd_pres_national <- row_count(filter(position_data_all, 
                                          section == 'national_presentations', end < 2021))
phd_pres_local    <- row_count(filter(position_data_all, 
                                          section == 'local_presentations', end < 2021))
phd_pres_outreach <- row_count(filter(position_data_all, 
                                          section == 'outreach_presentations', end < 2021))


postdoc_mentoring <- row_count(filter(position_data_all, 
                                          section == 'mentoring', end >= 2021))
phd_mentoring <- row_count(filter(position_data_all, 
                                          section == 'mentoring', end < 2021))
```


```{r, include=FALSE}
library(rvest)
# Get download values from zenodo
zenodo_metrics <- function(url = 'https://zenodo.org/records/7401113'){

  page_res <- read_html(url)
  page_sections <- page_res %>% html_elements("section")
  # page_sections[10][1] |> html_text2() |> print()
  page_sections <-  map(page_sections, html_text2)


  page_sections <- map(page_sections, function(x){
    x <- x |>
      stringr::str_replace_all('\n', '')
    out <- stringr::str_detect(x, '\\d*Views\\d*Downloads')
    if(out){
      out <- x
    }
  })
  metrics <- unlist(page_sections)

  metrics_views <- stringr::str_extract(metrics, '\\d*Views') |> stringr::str_extract('\\d*')
  metrics_downs <- stringr::str_extract(metrics, '\\d*Downloads') |> stringr::str_extract('\\d*')

  return(c(metrics_views, metrics_downs))
}

urls = c(
  # G3 Models
  'https://zenodo.org/records/7401113',

  # BLUP
  'https://zenodo.org/records/8367235',

  # Competition
  'https://zenodo.org/records/7830071',

  # G3 cleaned data
  'https://zenodo.org/records/6916775'

)

metrics <- map(urls, zenodo_metrics)

metrics <- data.frame(do.call(rbind, metrics))
names(metrics) <- c('Views', 'Downloads')

metrics_views <- sum(as.numeric(metrics$Views))
metrics_downs <- sum(as.numeric(metrics$Downloads))


format_txt_floor_nearest_5 <- function(val){
  if(val %% 5 >0){gt <- 'over **'}else{gt <- '**'}
  return(paste(c(gt, as.character(val - (val %% 5)), ' times**'), collapse = ''))
}
```




Aside
================================================================================
Contact Info {#contact}
--------------------------------------------------------------------------------
- `r icons::fontawesome$solid$envelope`<font size="1.5"> [daniel.r.kick@protonmail.com](daniel.r.kick@protonmail.com) </font>
<!-- - `r icons::fontawesome$solid$envelope` [hello@danielkick.com](hello@danielkick.com) -->
- `r icons::fontawesome$brands$linkedin` [daniel-kick-5a449b9a](https://www.linkedin.com/in/daniel-kick-5a449b9a/)
- `r icons::academicons$"google-scholar"` [Google Scholar](https://scholar.google.com/citations?user=5Sz0E-0AAAAJ&hl=en&oi=ao)
<!-- - `r icons::fontawesome$brands$orcid` [0000-0002-9002-1862](https://orcid.org/0000-0002-9002-1862) -->
- `r icons::fontawesome$brands$github` [github.com/danielkick](https://github.com/danielkick)
- `r icons::fontawesome$solid$link` [danielkick.com](https://www.danielkick.com)

## Education
### PhD: Biological Sciences

University of Missouri, Columbia, MO (2021)

<font size="-2">**Machine Learning Methods for Biomedical Informatics**, Quantitative Methods in the Life Sciences, and Grant Writing</font> 

<br>

### Bachelor of Science: Biology

Truman State University, Kirksville, MO (2015)

<font size="-2">Next Generation Sequence Data and Analysis, Bioinformatics
<!-- , ANOVA and Experimental Design, Non-Parametric Statistics -->

**Leadership role** in the biological honors society Tri-Beta</font>


<br>

### Technical Skills

 `r icons::fontawesome$brands$"r-project"`: **R **Programming  (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2017-01-29"), units = "weeks"))/52, 0)` years) and experience with <span style="color: DarkRed;">`tidyverse`, `lme4`, `caret`, `ggplot2`, `shiny`</span>, & package creation. 
 
 
 `r icons::fontawesome$brands$python`: **Python **Programming  (`r round(as.numeric(difftime(as.Date(format(Sys.time(), "%Y-%m-%d")), as.Date("2021-06-09"), units = "weeks"))/52, 0)` years) and experience with <span style="color:DarkRed;">`pandas`, `numpy`, `plotly`, `scikit-learn`, `keras`, `pytorch`</span>.
 

 `r icons::octicons$terminal`: **Miscellaneous **
 Experience with high performance computing (<span style="color:DarkRed;">`bash`, `slurm`</span>), virtual environments (<span style="color:DarkRed;">`conda`, `singularity`</span>), version control (<span style="color:DarkRed;">`git`, `GitHub`</span>), literate programming (<span style="color:DarkRed;">`Rmarkdown`, `Jupyter`</span>), crop growth modeling (<span style="color:DarkRed;">`APSIM Next Generation`</span>).


<!-- `r icons::octicons$person`: **Soft Skills** Adaptability | Problem Solving | Self-Directed Learning | Leadership | Mentorship  -->


Disclaimer {#disclaimer}
--------------------------------------------------------------------------------
For a pdf with

links scan here.



```{r, out.width="55%"}
knitr::include_graphics("./qr_gh_Resume_Resume.gif")
```
<!-- This resume was made with the R package [**pagedown**](https://github.com/rstudio/pagedown). -->
<!-- Last updated on `r Sys.Date()`. -->


Main
================================================================================

Daniel R. Kick, PhD {#title}
--------------------------------------------------------------------------------

- Experience with **statistics, programming, machine learning, and deep learning**.
- **Designed NIFA funded research plan** ([Grant 2023-67012-39485](https://portal.nifa.usda.gov/web/crisprojectpages/1030152-environmentally-aware-deep-learning-based-genomic-selection-and-management-optimization-for-maize-yield.html)).
- **Presented** to scientific and general audiences `r format_txt_floor_nearest_5(val = all_pres)`  since 2016. 
- **Led 4 teaching assistants** and **mentored 7 research students**.
- **Developed statistical tool** used by **>700 students** as of 2021.
- **Data and models** downloaded `r format_txt_floor_nearest_5(val = metrics_downs)` and viewed `r format_txt_floor_nearest_5(val = metrics_views)`.


```{r}
# paste(c("a", "b"), collapse = " | ")
```

<!-- Problem-solving |  Detail & Results-oriented | Statistical Analysis | Data Visualization | -->
<!-- Literate Programming | Technical & Non-technical Writer | Research & Development | Public Speaking | -->
<!-- Regulatory Affairs | Experimental Design | Method Development | Training, Mentorship, & Recruitment | -->
<!-- Collaborative | Open-mindedness | Growth mindset | Resourceful | Dependability | Adaptability | Self-starter | -->
<!-- Leadership | Effective Communicator | Fluorescent Microscopy | Microbiology | Pharmacology | Life Sciences -->

<!-- <span style="color: #003399;"> </span> -->
Professional and Research Experience {data-icon=laptop}
--------------------------------------------------------------------------------
### Research Geneticist

Jacob Washburn Lab, USDA-ARS

N/A

Present - 2021


- **Secured $225,000** to develop **"Environmentally Aware Deep Learning Based Genomic Selection And Management Optimization For Maize Yield"** from the National Institute of Food and Agriculture (NIFA) ([Grant 2023-67012-39485](https://portal.nifa.usda.gov/web/crisprojectpages/1030152-environmentally-aware-deep-learning-based-genomic-selection-and-management-optimization-for-maize-yield.html)).
- Employed **deep neural networks, machine learning models,** and **best linear unbiased predictors** to **improve corn yield prediction accuracy** in diverse environments.
- **Communicated** with stakeholders via **`r postdoc_pres_national+postdoc_pres_local+postdoc_pres_outreach` presentations** <font size="1">(`r postdoc_pres_national` national, `r postdoc_pres_local` regional, `r postdoc_pres_outreach` outreach)</font>. 
- **Mentored** `r postdoc_mentoring` students conducting a high throughput root phenotyping experiment. 
- **Created** and taught a **Python data visualization workshop** titled "Tools and Techniques for a Jupyter Based Scientific Workflow".
- Completed **Software Carpentries instructor certification**, taught **R for Reproducible Scientific Analysis**, and assisted in teaching **Data Management with SQL**.
- Designed and completed a professional development curriculum with the guidance of an industry scientist via the **Bayer-University Mentoring Program**.
- Served as a **panel member on "Next-Generation Omics"** at the 2022 University of Missouri Division of Biological Sciences Retreat.



### Graduate Researcher

David Schulz Lab, University of Missouri

N/A

2021 - 2015

- **Author on 6 publications**<font size="1">: 4 original research and 2 eLife Insight publications.</font>
- **Assessed the efficacy of machine learning models** to identify cell identity from mRNA and contig abundances. **Applied cluster estimation, hyperparameter tuning, unsupervised machine learning,** and **supervised machine learning**. Identified and learned needed skills primarily through self study. Collaborated with molecular biology project lead. (see *Northcutt^1^, Kick^1^, et al. 2019*).
- Defined research question and experiments. **Developed novel approaches to quantify changes in cell activity**.
- **Collaborated with electrophysiologists**, assisting with data analysis.
- **Collaborated with computational neuroscientists**, contributing domain expertise.
- **Mentored `r phd_mentoring` students** and oversaw their projects. 
- Communicated results through **`r phd_pres_national+phd_pres_local+phd_pres_outreach` presentations** <font size="1">(`r phd_pres_national` national, `r phd_pres_local` regional, `r phd_pres_outreach` outreach)</font>. 
- Served as a **peer mentor of 3 PhD students** in use of R for reproducible data analysis, created internal documents on same.

<!-- (https://www.pnas.org/doi/10.1073/pnas.1911413116) -->
<!-- [2](https://www.jneurosci.org/content/42/35/6751) -->
<!-- [3](https://mospace.umsystem.edu/xmlui/handle/10355/85790) -->

### Lead Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2021 - 2020

- **Developed statistics web application used by more than 700 students** as of 2021 with **shiny** ([source](https://github.com/danielkick/MultiMethodPhysApp), [deployed](https://danielkick.shinyapps.io/BioSc3700_1)) for data visualization, testing assumptions, and fitting **frequentist**, **non-parametric**, and **Bayesian** models.
- Led 4 Teaching Assistants and coordinated adaptation of lab curriculum to be fully online due to COVID-19 pandemic.
- **Mentored** next Lead Teaching Assistant, created documentation on best practices.

### Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2020 - 2018

- Updated and refined curriculum, delivered lectures and ensured experiments were conducted safely, and modeled student grade distributions to identify and adjust for differences in grading.


### Curriculum Consultant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2018

- Redesigned course material to incorporate primary literature and data analysis.


### Teaching Assistant, Animal Physiology Lab

Biological Sciences, University of Missouri

N/A

2016 - 2015

- Delivered weekly lectures, ensured experiments were conducted safely, provided timely feedback on assignments.


### Undergradute Researcher

University of Missouri, University of Connecticut, and Truman State University

N/A

2013 - 2015

- Designed a hydroponic system for maize root phenotyping <font size="1">--(2014-2015)</font>, Quantified retinal minor splicisome expression using immunohistochemistry <font size="1">-- (2014)</font>, Measured effectiveness of oligonucleotide treatment for spinal muscular atrophy in mice <font size="1">-- (2013)</font>.


Honors and Awards (`r resume_frac(get_section = 'honors')`) {data-icon=award}
--------------------------------------------------------------------------------
```{r}
position_data %>% print_section('honors')
```



<!-- ### J. Perry Gustafson Award for Outstanding Graduate Research in the Life Sciences -->

<!-- [This award](https://randall-travel-award.missouri.edu/gustafson-award/) is granted for the quality of a student's independent research and academic achievements. Recipients receive a \$2,000 award. -->

<!-- N/A -->

<!-- 2019 -->


<!-- ### National Institutes of Health T32 Training Grant Recipient -->

<!-- This fellowship provides a \$27,000 yearly stipend and travel awards of \$750.   -->

<!-- N/A -->

<!-- 2018 - 2016 -->


<!-- ### Cum Laude & President's Recognition, Truman State University -->

<!-- N/A -->

<!-- N/A -->

<!-- 2015 -->






 <!-- `r resume_frac(get_section = 'academic_articles_review')` in Review, -->



Selected Publications (Showing `r resume_frac(get_section = 'academic_articles')` Published, 0/`r resume_frac(get_section = 'academic_articles_prep', half = T)` in Prep.) {data-icon=flask}
--------------------------------------------------------------------------------
```{r}
# position_data %>% print_section('academic_articles')
```


### Ensemble of Best Linear Unbiased Predictor, Machine Learning, and Deep Learning Models Predict Maize Yield Better Than Each Model Alone

<font size="1">***Daniel R. Kick***, Jacob D. Washburn [_in Silico_ Plants](https://doi.org/10.1093/insilicoplants/diad015)</font>

N/A

2023

### Yield Prediction Through Integration of Genetic, Environment, and Management Data Through Deep Learning

<font size="1">***Daniel R. Kick***, Jason G. Wallace, James C. Schnable, Judith M. Kolkman, Baris Alaca, Timothy M. Beissinger, David Ertl, Sherry Flint-Garcia, Joseph L. Gage, Candice N. Hirsch, Joseph E. Knoll, Natalia de Leon, Dayane C. Lima, Danilo Moreta, Maninder P. Singh, Teclemariam Weldekidan, Jacob D. Washburn [G3: Genes, Genomes, Genetics](https://academic.oup.com/g3journal/advance-article/doi/10.1093/g3journal/jkad006/6982634)</font>

N/A

2023


### Timing dependent potentiation and depression of electrical synapses contributes to network stability in the crustacean cardiac ganglion

<font size="1">***Daniel R. Kick*** and David J. Schulz [The Journal of Neuroscience](https://www.jneurosci.org/content/42/35/6751)</font>

N/A

2022
<!-- [Code](https://github.com/danielkick/ElectricalSynapsePotentiation) -->


### Molecular profiling of single neurons of known identity in two ganglia from the crab Cancer borealis

<font size="1">Adam J. Northcutt^1^, ***Daniel R. Kick^1^***, Adriane G. Otopalik, Benjamin M. Goetz, Rayna M. Harris, Joseph M. Santin, Hans A. Hofmann, Eve Marder, and David J. Schulz (***^1^ denotes co-first authorship***) [Proceedings of the National Academy of Sciences](https://www.pnas.org/doi/10.1073/pnas.1911413116)</font>

N/A

2019


<!-- [Code](https://github.com/danielkick/STGmRNACellClassification) -->

